clients/vscode/package.json

```diff
@@ -400,6 +419,11 @@
         "key": "ctrl+l",
         "mac": "cmd+l",
         "when": "!editorHasSelection && tabby.chatEnabled && !tabby.chatViewVisible"
+      },
+      {
+        "command": "tabby.chat.edit.editNLOutline",
+        "key": "ctrl+y",
+        "mac": "cmd+y"
```

Comment:
when = tabby.chatOutlineEnabled

Reply:
I believe that performing a context var check separately is inconvenient. Therefore, I implemented this check within this command. It should be related to the `editNLOutline` function in `/src/outline/index.ts`.